admin-dev/themes/default/scss/controllers/_login.scss

```diff
@@ -77,7 +76,6 @@
 
   .front,
   .back {
-    position: absolute;
     top: 0;
```

Comment:
Please remove this too

Reply:
And the left property, just under

Reply:
And the transform style is causing some blurry effect, you can remove every transform-style, I think it should work without it (and verify of course)

Reply:
@NeOMakinG I don't find any "left" property under ...

Reply:
done

Reply:
@Amoifr Seems like the top hasn't been removed, this property is useless since we removed the position: absolute

Is there a reason why you didn't remove it?